### PR TITLE
Issue/13

### PR DESCRIPTION
**Pull request by Knoxie**
Populate list on TheMovieDB.org for importing into other software (radarr/couch/etc)

### CODE REVIEW

File: src/main/java/com/jasonhhouse/Gaps/GapsApplication.java [hunk 1]
```diff
@@ -84,6 +76,109 @@ public void run(String... args) {
...
+            logger.error("Unable to create session id: " + e.getMessage());
+            return;
+        }
+
+
+        // Write the session id to Yaml file
+        // TheMovieDB_session: 888d4d8d35f531d4c8c8ae1e66e4d167be5972f0
+        // Create list, or use
+
+        /**
```
Comment (JasonHHouse) [inline src/main/java/com/jasonhhouse/Gaps/GapsApplication.java]: Any need for this commented code anymore?

File: src/main/java/com/jasonhhouse/Gaps/GapsApplication.java [hunk 2]
```diff
@@ -84,6 +76,109 @@ public void run(String... args) {
...
+            logger.info("Click the link below to authorize TMDB list access: " + "https://www.themoviedb.org/authenticate/" + request_token);
+            logger.info("Press enter to continue");
+            logger.info("############################################");
+            System.in.read();
+        } catch (Exception e) {
+            logger.error("Unable to authenticate tmdb, and add movies to list. ");
+            logger.error(e.getMessage());
+            return;
+        }
+        // approve https://www.themoviedb.org/authenticate/ccf26e747f3ae7cd40ce989a3caddd1445f7e976/
```
Comment (JasonHHouse) [inline src/main/java/com/jasonhhouse/Gaps/GapsApplication.java]: Do we want this hardcoded, what looks like a key in here?

File: src/main/java/com/jasonhhouse/Gaps/GapsApplication.java [hunk 3]
```diff
@@ -10,9 +10,7 @@
 
 package com.jasonhhouse.Gaps;
 
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
+import okhttp3.*;
```
Comment (JasonHHouse) [inline src/main/java/com/jasonhhouse/Gaps/GapsApplication.java]: No * imports

File: src/main/java/com/jasonhhouse/Gaps/GapsApplication.java [hunk 4]
```diff
@@ -34,13 +32,7 @@
...
 import javax.xml.xpath.XPathExpressionException;
 import javax.xml.xpath.XPathFactory;
-import java.io.ByteArrayInputStream;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.UnsupportedEncodingException;
+import java.io.*;
```
Comment (JasonHHouse) [inline src/main/java/com/jasonhhouse/Gaps/GapsApplication.java]: No * imports

File: src/main/java/com/jasonhhouse/Gaps/GapsApplication.java [hunk 5]
```diff
@@ -84,6 +76,109 @@ public void run(String... args) {
...
+            request_token = responseJson.getString("request_token");
+
+            // Have user click link:
+            logger.info("############################################");
+            logger.info("Click the link below to authorize TMDB list access: " + "https://www.themoviedb.org/authenticate/" + request_token);
+            logger.info("Press enter to continue");
+            logger.info("############################################");
+            System.in.read();
+        } catch (Exception e) {
+            logger.error("Unable to authenticate tmdb, and add movies to list. ");
```
Comment (JasonHHouse) [inline src/main/java/com/jasonhhouse/Gaps/GapsApplication.java]: You can change this to be a single line.
logger.error("Unable to authenticate tmdb, and add movies to list.", e);

**Comment by Knoxie** [comment]
@JasonHHouse Ready for another review/merge

**Pull request merged**